products/resourcemanager/terraform.yaml

```diff
@@ -16,6 +16,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   Project: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
   Lien: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
```

Comment:
Lien doesn't appear to be an async resource.

Reply:
Good call. I'll fix that.